### PR TITLE
feat(container): add container_token resource

### LINKED ISSUE #1518: serverless: container token
Container api deprecated IssueJwt and created a CreateToken method.
We should make the change if needed

### PR DESCRIPTION
**Pull request by Codelax**
Close #1518

### CODE REVIEW

File: scaleway/resource_container_token.go [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+			"expires_at": {
+				Type:             schema.TypeString,
+				Optional:         true,
+				ForceNew:         true,
+				ValidateDiagFunc: validateDate(),
+				DiffSuppressFunc: diffSuppressFuncTimeRFC3339,
+			},
+			"token": {
+				Type:     schema.TypeString,
+				Computed: true,
```
Comment (remyleone) [inline scaleway/resource_container_token.go:49]: Should we add `Sensitive:   true` here ?

File: scaleway/resource_container_token.go [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+			return nil
+		}
+		return diag.FromErr(err)
+	}
+
+	_ = d.Set("container_id", flattenStringPtr(token.ContainerID))
+	_ = d.Set("namespace_id", flattenStringPtr(token.NamespaceID))
+	_ = d.Set("description", flattenStringPtr(token.Description))
+	_ = d.Set("expires_at", flattenTime(token.ExpiresAt))
+	_ = d.Set("token", token.Token)
```
Comment (remyleone) [inline scaleway/resource_container_token.go:100]: it is sensitive

File: scaleway/resource_container_token.go [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+	}
+}
+
+func resourceScalewayContainerTokenCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	api, region, err := containerAPIWithRegion(d, meta)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	token, err := api.CreateToken(&container.CreateTokenRequest{
```
Comment (Monitob) [inline scaleway/resource_container_token.go:64]: Please add the scw Context

File: scaleway/resource_container_token.go [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+	return resourceScalewayContainerTokenRead(ctx, d, meta)
+}
+
+func resourceScalewayContainerTokenRead(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	api, region, ID, err := containerAPIWithRegionAndID(meta, d.Id())
+	if err != nil {
+		return diag.FromErr(err)
+	}
+	token, err := api.GetToken(&container.GetTokenRequest{
```
Comment (Monitob) [inline scaleway/resource_container_token.go:85]: context missing

File: scaleway/resource_container_token.go [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+	return nil
+}
+
+func resourceScalewayContainerTokenDelete(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	api, region, ID, err := containerAPIWithRegionAndID(meta, d.Id())
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	_, err = api.DeleteToken(&container.DeleteTokenRequest{
```
Comment (Monitob) [inline scaleway/resource_container_token.go:112]: context missing

**Review by Monitob — CHANGES_REQUESTED** [review]

**Review by Monitob — APPROVED** [review]

**Review by remyleone — APPROVED** [review]

**Pull request merged**